src/pluggy.ts

```diff
@@ -30,9 +30,10 @@ class Pluggy extends BaseApi {
    * @throws {AxiosError<ErrorResponse>} status 403 if user is unauthorized
    */
   async fetchConnectors(
-    options: ConnectorFilters = {}
+    options: ConnectorFilters = {},
+    includeHealth: boolean = false
```

Comment:
it's unnecesary to specify the type when we are also initialziing the value

Reply:
Ok!
Also: how about defining this new parameter `includeHealth` as an object instead? This way the usage would be a bit more clearer and scalable if we want to add more options.
For example instead of doing 
```ts
  const response = await client.fetchConnectors(
    {
      sandbox: true,
    },
    true
  )
```
Which is a bit weird (what that `true` means - at first sight?), we could do instead:
```ts
  const response = await client.fetchConnectors(
    {
      sandbox: true,
    },
    { includeHealth: true }
  )
````
With also the gained flexibilty of being able to add more options in the future.

Reply:
I'll leave that for future refactors when having new options!